src/chrome/LabeledStats.tsx

```diff
@@ -30,40 +34,78 @@ export interface LabeledStatsProps {
   uppercase?: boolean
 }
 
+export function getStats (data: Structure): any[] {
+  return [
+    { 'label': 'format', 'value': data.format ? data.format.toUpperCase() : 'unknown' },
+    { 'label': 'body size', 'value': data.length ? fileSize(data.length) : '—' },
+    { 'label': 'entries', 'value': abbreviateNumber(data.entries) || '—' },
+    { 'label': 'errors', 'value': data.errCount ? abbreviateNumber(data.errCount) : '—' },
+    { 'label': 'depth', 'value': data.depth || '—' }
+  ]
+}
```

Comment:
the stats prop is intended to be generic. Move this into `/src/qri/dataset.ts`, somewhere near the declaration of the structure component, restore the `Stat` interface, and pass the result of calling `getStats` to the component

Reply:
LabeledStats is rendering a tooltip that contains info from `structure`, so we need to pass structure in as a prop and call `getProps()` from within this component.  Either that or we need a separate component to render the tooltip that will select structure on its own, which feels wrong.

I've moved the `Stat` interface to `/qri/dataset.ts` and made sure that `getStats()` is returning `Stat[]`